CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # These are the default owners for the whole content of the `luigi` repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
-* @alexandra-simeonova @hardl @johannesdoberer @maxmarkus @ndricimrr @stanleychh @ulianamunich @antiheld @legteodav
+* @alexandra-simeonova @hardl @johannesdoberer @maxmarkus @ndricimrr @stanleychh @ulianamunich @antiheld @legteodav @wdoberschuetz
```

Comment:
```suggestion
* @alexandra-simeonova @hardl @johannesdoberer @ndricimrr @stanleychh @ulianamunich @antiheld @wdoberschuetz
```

Reply:
btw who is "antiheld" 🤔  ?

Reply:
Removed maxmarkus, legteodav and antiheld.